uptane-standard.md

```diff
@@ -807,19 +808,20 @@ Full verification MAY be performed by either primary or secondary ECUs. The proc
-4. Download and check the Timestamp metadata file from the Director repository, following the procedure in {{check_timestamp}}.
-5. Download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
-6. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
-7. Download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
-8. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
-9. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
-10. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
+1. Load and verify the current time, or the most recent securely attested time.
+1. Download and check the Root metadata file from the Director repository, following the procedure in {{check_root}}.
+1. Download and check the Timestamp metadata file from the Director repository, following the procedure in {{check_timestamp}}.
+1. Download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
+1. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
+1. Download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
+1. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
+1. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
+1. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
+1. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
```

Comment:
```suggestion
10. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
```